stipendien/deutschland.html

```diff
@@ -0,0 +1,36 @@
+---
+title: Deutschlandstipendien
+subtitle: Hier gibt es Informationen zu den Deutschlandstipendien.
+layout: page-custom
+---
+<div class="card">
+  <div class="card-body">
+    <p>
```

Comment:
and still use markdown here (instead of explicitly writing all these html tags everywhere)